config/application.rb

```diff
@@ -58,11 +58,18 @@ class Application < Rails::Application
     # we do here instead of using initializers because of a Rails 5.1 vs
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
-    # This is fixed in Rails 5.2
-    config.load_defaults 5.1
+    config.load_defaults 6.0
     config.active_record.belongs_to_required_by_default = false
     config.active_record.include_root_in_json = true
     # Make `form_with` generate non-remote forms. Defaults true in Rails 5.1 to 6.0
+    config.action_view.form_with_generates_remote_forms = false
+    # Make Ruby preserve the timezone of the receiver when calling `to_time`.
+    config.active_support.to_time_preserves_timezone = false
+    config.action_dispatch.use_authenticated_cookie_encryption = false
+    config.active_support.use_authenticated_message_encryption = false
```

Comment:
I assume these two keep old defaults, correct?

Reply:
hmm, I see that these were already true in 5.2 https://guides.rubyonrails.org/configuring.html#default-values-for-target-version-5-2

Or do I miss where we changed them before?

They seem to be safe to update to `true` though. I guess only cookies will be invalidated, but in the past we considered that alright (for the FIPS update we had a similar situation).

Reply:
Those two where introduced in `5.2`, but we were using `5.1` defaults, so they were not set in porta. Now we use `6.0` defaults, those two are set to `true`, so I need to manually set them back to `false` to keep old behavior. Another question is if we want to keep the old behavior. I think we probably don't. WDTY?

Reply:
I think we can use the newer defaults for `use_authenticated_cookie_encryption` and `use_authenticated_message_encryption`, and keep the overridden behavior for `to_time_preserves_timezone`

Reply:
agreed new defaults should be fine, whether to do now or after, is another question I have no hard opinion about

Reply:
Ok, I removed those two lines.